src/ports/postgres/modules/convex/test/unit_tests/test_mlp_igd.py_in

```diff
@@ -126,10 +126,11 @@ class MLPMiniBatchTestCase(unittest.TestCase):
                                                 'dependent_varname': 'value',
                                                 'class_values': 'regression',
                                                 'grouping_cols': 'value',
+                                                'dependent_vartype': 'value',
                                                 'foo': 'bar'}]
         self.module = self.subject.MLPMinibatchPreProcessor("input")
         self.assertTrue(self.module.preprocessed_summary_dict)
-        self.assertEqual(5, len(self.module.preprocessed_summary_dict))
+        self.assertEqual(6, len(self.module.preprocessed_summary_dict))
```

Comment:
Let's put this as a separate commit (can be pushed to master already) since it's unrelated to this PR.